autoextract/aio/client.py

```diff
@@ -173,10 +177,17 @@ async def request_raw(query: Query,
     if agg_stats is None:
         agg_stats = AggStats()  # dummy stats, to simplify code
 
+    if max_query_error_retries and not handle_retries:
+        logger.warning(
```

Comment:
I saw this message in another part of the code. It will be shown twice when invoking the cmd. Consider leaving only one, probably to show it only in the cmd?

Reply:
Yes, but don't you want me to leave it in the most inner part of the code so that it's logged whether user is consuming our library API or using our command-line tool?

Reply:
The `--disable-retries` command-line argument was removed after a comment from @kmike. This log was also removed in the same occasion.

Reply:
IMO is better not show the warning at all. For a developer who is using the api, it can be handy to have a default `max_query_error_retries` and then use the flag `handle_retries` to enable/disable the whole retries at all. I think is enough to have it documented in the function docstring

Reply:
Why is logger.warning used here, not warnings.warn? I.e. why is it better to show this warning every time the function is called, not just once?

Reply:
No particular reason. I think `warnings.warn` is a better option here. Refactored.